File: 백준/Silver/1978. 소수 찾기/소수 찾기.py
n = int(input())
arr = list(map(int, input().split()))
count = 0

for i in arr:  # arr리스트 요소
    for j in range(2, i + 1):  # range(2,4)
        if i % j == 0:
            if i == j:
                count += 1
            break
print(count)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.util.Arrays;

import java.util.Scanner;

public class Main {
    static boolean isPrime(int n) {
        if (n < 2) {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        scan.nextInt();
        scan.nextLine();
        Main obj = new Main();

        int[] arr = Arrays.stream(scan.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int count = 0;
        for(int k : arr){
            if (obj.isPrime(k)){
                count++;
            }
        }
        System.out.println(count);

    }
}
